packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -274,15 +274,18 @@ type InsertionMode = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7;
 export type FormatContext = {
   insertionMode: InsertionMode, // root/svg/html/mathml/table
   selectedValue: null | string | Array<string>, // the selected value(s) inside a <select>, or null outside <select>
+  allowResources: boolean,
```

Comment:
For the nesting warnings we name these by the mechanism `buttonTagInScope`.

For `resourceFormOnly` you named it based on its purpose down the tree, and the same here.

These two patterns are not quite compatible.

Might be better sticking to what info the outer scope provides if it can be reused instead of creating a specific purpose that we don't yet know if it'll generalize. It's like an early abstraction.

`noscriptTagInScope` maybe?

Reply:
Yeah that makes sense. I'll update this PR for allowResources and follow up tomorrow with one for resourceFormOnly